sandhisplitter/tests/test_model.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from sandhisplitter.model import Model
+from testtools import TestCase
+from sandhisplitter.util import extract
+
+
+class TestModel(TestCase):
+    def setUp(self):
+        super(TestModel, self).setUp()
+        self.testModel = Model(depth=3, skip=1)
+
+    def test_load(self):
+        entries = [
+                u'വിഡ്‌ഢിയുണ്ടെങ്കില്‍=വിഡ്‌ഢി+ഉണ്ട്+എങ്കില്‍|7,11',
+                u'അകല്‍ച്ചയുടെസ്ഥാനത്തു്=അകല്‍ച്ചയുടെ+സ്ഥാനത്തു്|10',
```

Comment:
Can we place test cases in a separate file under sandhisplitter/tests/resources and load it in setUp function?. If there are multiple tests we can utilize same entries in all tests